package/src/components/Channel/Channel.tsx

```diff
@@ -1219,7 +1226,7 @@ const ChannelWithContext = <
       loadMoreThreadFinished(updatedHasMore, updatedThreadMessages);
       const { messages } = await channel.getMessagesById([parentID]);
       const [threadMessage] = messages;
-      if (threadMessage) {
+      if (threadMessage && !thread.activate) {
```

Comment:
I'm sure we had a talk about this but I can't remember - why are you checking for `activate` here? Would check for existence of thread instance suffice? If not, let's check whether it's actually an instance of the Thread class. :)

```suggestion
      if (threadMessage && !(thread instanceof Thread)) {
```

Reply:
Oops, I guess I forgot to move this ! Great catch, thanks !